modules/puppet_proxy/initializer.rb

```diff
@@ -1,4 +1,4 @@
-require 'puppet'
+require 'puppet' if RUBY_PLATFORM =~ /ruby/
```

Comment:
Again, I don't think this is useful: the module isn't going to work without this dependency, this will only hide the error which otherwise would disable the module on startup.